tests/filecheck/dialects/riscv/riscv_assembly_emission.mlir

```diff
@@ -0,0 +1,11 @@
+// RUN: xdsl-opt -p riscv-allocate-registers -t riscv-asm %s | filecheck %s
+
+"builtin.module"() ({
+  %0 = "riscv.li"() {"immediate" = 6 : i32} : () -> !riscv.reg<>
+  %1 = "riscv.li"() {"immediate" = 5 : i32} : () -> !riscv.reg<>
+  %2 = "riscv.add"(%0, %1) : (!riscv.reg<>, !riscv.reg<>) -> !riscv.reg<>
+}) : () -> ()
+
+// CHECK:      li j0, 6
+// CHECK-NEXT: li j1, 5
+// CHECK-NEXT: add j2, j0, j1
```

Comment:
```suggestion
// CHECK-NEXT: add j2, j0, j1

```
There's a vscode setting to add newlines, also I think there's a pre-commit hook but I'm not sure if that needs to be set up explicitly

Reply:
I'll look into this! thanks for the suggestion ;) I'll force push really quickly